cfme/fixtures/v2v.py

```diff
@@ -165,7 +165,7 @@ def get_vm(request, appliance, second_provider, template, datastore='nfs'):
     request.addfinalizer(lambda: vm_obj.cleanup_on_provider())
     return vm_obj
 
-
+@pytest.fixture(scope='function')
```

Comment:
Hey Shveta, can you help me understand why are we making this as a fixture? Ytale had made it more of like a helper function when he designed it.

Reply:
Hey Kedar , yes I needed a simple form_data to fill and edit and didn't want to make the test case complicated with parametrization just for editing hence used form_data

Reply:
That doesn't make it clear to me why it was marked as fixture? If you just needed the form data, you could have called it as a function within your test call phase?